### ISSUE TITLE
Missing description for the Example Object

### ISSUE THREAD

**Issue by blendsdk** [OAI/OpenAPI-Specification#2160]
Hi,
The Example Object in v3.1.0-dev (and 3.0.x) is missing a description. Not that this is urgent but something along these lines would be nice:

```
#### <a name="exampleObject"></a>Example Object
Allows sharing an example to demonstrate the usage of properties, parameters and objects within OpenAPI
```

Regards,
Gevik

**Comment by MikeRalphson** [comment]
Thanks - it would be nice to fix this. Do you have time to create a PR against `v3.1.0-dev`?

**Comment by PradipSable07** [comment]
Hello sir, 
i am interested in this problem, can i do this?
@MikeRalphson @blendsdk @danhorst @duncanbeevers

**Comment by duncanbeevers** [comment]
@PradipSable07 I think a PR would be great, though I don't have the power to approve one.

**Comment by handrews** [comment]
PR merged for 3.0.4 and ported to 3.1.1 via PR #3921!
